john_tucker_miller/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "john_tucker_miller",
+  "version": "0.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "BSD-2-Clause",
+  "devDependencies": {
+    "body-parser": "~1.14.1",
+    "express": "~4.13.3",
```

Comment:
body-parser and express are dependencies--your project needs them in order to run.